go/vt/sqlparser/ast.go

```diff
@@ -996,7 +996,8 @@ const (
 
 // Show represents a show statement.
 type Show struct {
-	Type string
+	Type   string
+	Target ColIdent
```

Comment:
`ColIdent` is case-insensitive. So, it's better to use `TableIdent` instead, which is case-sensitive.